nemo/collections/asr/parts/dataset.py

```diff
@@ -152,7 +153,9 @@ def __init__(
     def __getitem__(self, index):
         sample = self.collection[index]
         if self.load_audio:
-            features = self.featurizer.process(sample.audio_file, offset=0, duration=sample.duration, trim=self.trim,)
+            features = self.featurizer.process(
+                expanduser(sample.audio_file), offset=0, duration=sample.duration, trim=self.trim,
```

Comment:
Can you move this into the manifest processing instead of getitem? That way you do it once for the entire dataset rather than doing it everytime you get an item from the dataloader

Reply:
you mean into the futurizer?

Reply:
I mean here: https://github.com/NVIDIA/NeMo/blob/dev-jasper-unjenkinsing/nemo/collections/asr/parts/manifest.py#L84

Reply:
*WaveformFeaturizer*?

Reply:
@blisc I started to trace that, but this goes deeper into *uncharted territory* and classes like `AudioSegment` etc.

Please keep in mind that my goal is to refactor a single JASPER example, so IMO this goes waaaaay beyond the original scope and learning what really is down there will take take, pulling me from my main task. 

As this works for me, I am giving up with propagating that further. That does not means that it shouldn't be done - please go ahead if you like. :)

Reply:
I am still completely against using expanduser(), but I added the fix